src/Avalonia.Themes.Fluent/CaptionButtons.xaml

```diff
@@ -0,0 +1,70 @@
+<Styles xmlns="https://github.com/avaloniaui" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+  <Style Selector="CaptionButtons">
+    <Setter Property="Foreground" Value="{DynamicResource ThemeForegroundBrush}"/>
+    <Setter Property="Canvas.Right" Value="0" />
+    <Setter Property="MaxHeight" Value="30" />
+    <Setter Property="Template">
+      <ControlTemplate>
+        <StackPanel Spacing="2" Margin="0 0 7 0" VerticalAlignment="Stretch" TextBlock.FontSize="10" Orientation="Horizontal">
+          <StackPanel.Styles>
+            <Style Selector="Panel">
+              <Setter Property="Width" Value="45" />
+              <Setter Property="Background" Value="Transparent" />
+            </Style>
+            <Style Selector="Panel:pointerover">
+              <Setter Property="Background" Value="#7F7f7f7f" />
+            </Style>
+            <Style Selector="Panel#PART_CloseButton:pointerover">
+              <Setter Property="Background" Value="#7FFF0000" />
+            </Style>
+            <Style Selector="Viewbox">
+              <Setter Property="Width" Value="11" />
+              <Setter Property="Margin" Value="2" />
+            </Style>
+          </StackPanel.Styles>
+          <Panel x:Name="PART_FullScreenButton">
+            <Viewbox>
+              <Path Stretch="UniformToFill" Fill="{TemplateBinding Foreground}" />
+            </Viewbox>
+          </Panel>
+
+          <Panel x:Name="PART_MinimiseButton">
+            <Viewbox>
+              <Path Stretch="UniformToFill" Fill="{TemplateBinding Foreground}" Data="M2048 1229v-205h-2048v205h2048z" />
+            </Viewbox>
+          </Panel>
+
+          <Panel x:Name="PART_RestoreButton">
+            <Viewbox>
+              <Viewbox.RenderTransform>
+                <RotateTransform Angle="-90" />
```

Comment:
Why is this needed? Restore button path seems to be a square.

Reply:
![image](https://user-images.githubusercontent.com/4672627/86381141-5ad64f00-bc63-11ea-8574-0f7c99361cd2.png)
not when maximised.